.github/workflows/github-commands-comment.yml

```diff
@@ -26,7 +26,7 @@ jobs:
               <p>
 
               Just comment with:
-              - `run` `docs-build` : Re-trigger the docs validation. (use unformatted text in the comment!)
+              - \`run\` \`docs-build\` : Re-trigger the docs validation. (use unformatted text in the comment!)
```

Comment:
is there a reason we aren't using the same yml file as in e.g. https://github.com/elastic/apm-agent-go/pull/1721/files ?

Reply:
I created this initially, without using the https://github.com/elastic/oblt-actions/tree/main/elastic/github-commands . Then I realised that using the composite github actions was easier so we can change behaviour easily in one place

Reply:
Actually I changed it now:
[`e0a358a` (#1551)](https://github.com/elastic/apm-agent-ruby/pull/1551/commits/e0a358a6a979b792648a8470d3fdb75714550784)